modules/social_features/social_album/social_album.install

```diff
@@ -307,3 +324,51 @@ function social_album_update_11401(): string {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Grant group albums permissions for anonymous users.
+ */
+function social_album_update_11402(): void {
+  /** @var \Drupal\group\Entity\GroupRoleInterface $group_role_anonymous */
+  $group_role_anonymous = \Drupal::entityTypeManager()->getStorage('group_role')->load('flexible_group-anonymous');
+  $group_role_anonymous->grantPermissions(['view group_node:album content', 'view group_node:album entity'])->save();
+}
+
+/**
+ * Grant group albums permissions for group outsiders.
+ */
+function social_album_update_11403(): void {
+  /** @var \Drupal\group\Entity\GroupRoleInterface $group_role_outsider */
+  $group_role_outsider = \Drupal::entityTypeManager()->getStorage('group_role')->load('flexible_group-outsider');
+  $group_role_outsider->grantPermissions(['view group_node:album content', 'view group_node:album entity'])->save();
+
+  /** @var \Drupal\group\Entity\GroupRoleInterface $group_role_member */
+  $group_role_member = \Drupal::entityTypeManager()->getStorage('group_role')->load('flexible_group-member');
+  $group_role_member->grantPermissions(['view group_node:album content', 'view group_node:album entity'])->save();
+}
+
+/**
+ * Update Album permissions.
+ */
+function social_album_update_11404(): void {
+  user_role_grant_permissions(
+    RoleInterface::ANONYMOUS_ID,
+    [
+      'view node.album.field_content_visibility:public content',
+    ]
+  );
+
+  user_role_grant_permissions(
+    'verified',
+    [
+      'view node.album.field_content_visibility:community content',
```

Comment:
I see that this was already added in https://github.com/goalgorilla/open_social/blob/main/modules/social_features/social_album/social_album.install#L28, did few a reason why it was missed being added ?